vizro-core/docs/pages/user-guides/assets.md

```diff
@@ -33,6 +33,14 @@ For more information, see the [Dash documentation](https://dash.plotly.com/exter
 To overwrite any global CSS properties of existing components, target the right CSS property and place your CSS files in the `assets` folder. This will overwrite any existing defaults for that CSS property.
 For reference, see the [Vizro CSS files](https://github.com/mckinsey/vizro/tree/main/vizro-core/src/vizro/static/css).
 
+CSS properties will be applied with the last served file taking precedence. The order of serving is:
+
+1. Dash built-in stylesheets
+2. Vizro built-in stylesheets
+3. User assets folder stylesheets
+
+Within each of these categories, individual files are served in alphanumeric order.
+
```

Comment:
Let me remove it to the bottom then. I wouldn't remove it just yet, because it does become relevant when it comes to CSS specificity. That would fit well into our separate "Intro to CSS page with commonly asked questions" that does not exist yet 😄